### DISCUSSION TITLE
More parameters for Drift Detectors

### DISCUSSION THREAD

**Discussion by TawabG** [online-ml/river#869] [Q&A]
Hi all!

I have a question related to the different drift detectors available by River.
Let's take for example https://riverml.xyz/latest/api/drift/EDDM/

As can be seen from the summary, Alpha and Beta are set at 0.95 and 0.90 respectively.

In the code (https://github.com/online-ml/river/blob/main/river/drift/eddm.py) these are set as 
` 
FDDM_OUTCONTROL = 0.9
FDDM_WARNING = 0.95
FDDM_MIN_NUM_INSTANCES = 30
`
Is it possible to add these as parameters for the function?

**Comment by MaxHalford** [comment]
Hey! For sure, that's definitely possible. Would you be inclined to contribute a pull request? :)

**Reply by TawabG** [reply]
Sure😃 I'm going to work on it now!

**Reply by TawabG** [reply]
@MaxHalford Can you have a look at https://github.com/online-ml/river/pull/870
